cclib/parser/psi4parser.py

```diff
@@ -718,9 +729,13 @@ def get_normalization_factor(exp, lx, ly, lz):
         #
         # Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
         #
+        # OR
+        #
+        # Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]
+        #
         if "Properties will be evaluated at" in line.strip():
             self.origin = numpy.array([float(x.strip(',')) for x in line.split()[-4:-1]])
-            assert line.split()[-1] == "Bohr"
+            assert line.split()[-1] in ["Bohr", "[a0]"]

```

Comment:
So if the units are angstroms here, do you need to do the conversion on the line below?

Is the regression file you added in cclib/cclib-data#95 represents this case, you can consider adding an appropriate regression test in `regression.py` (in the main repo) and check that the parsed value is in the right units. The tricky thing is to name the regression test function correctly, and the file has gotten really big, so I can take care of that if needed.

Reply:
I assumed that `a0` would still be Bohr, while I would expect something like `ang` to be Angstrom

https://en.wikipedia.org/wiki/Bohr_radius